cluster/src/test/scala/org/apache/spark/sql/MiscTest.scala

```diff
@@ -88,4 +88,30 @@ class MiscTest extends SnappyFunSuite with Logging {
       snc.sql(s"set schema app")
     }
   }
+
+  test("SNAP-2440") {
```

Comment:
Can you add  negative tests for wrong table name, invalid function name etc.

Reply:
It is there. Please check testt.* negative case.